app/admin/user.rb

```diff
@@ -0,0 +1,32 @@
+ActiveAdmin.register User do
+  permit_params :name, :surname, :patronymic, :email, :phone, :address, :encrypted_password
+
+  menu label: 'Пользователи'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.